modules/rc_fax/contact.php

```diff
@@ -0,0 +1,162 @@
+<?php
+/**
+ * Fax SMS Module Member
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Jerry Padgett <sjpadgett@gmail.com>
+ * @copyright Copyright (c) 2018-2019 Jerry Padgett <sjpadgett@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once("./libs/controller/oeFaxSMSClient.php");
+
+use OpenEMR\Core\Header;
+
+// kick off app endpoints controller
+$clientApp = new oeFaxSMSClient();
+$logged_in = $clientApp->authenticate();
+$the_file = $clientApp->getRequest('file');
+$isContent = $clientApp->getRequest('isContent');
+$isDoc = $clientApp->getRequest('isDocuments');
+$file_name = pathinfo($the_file, PATHINFO_BASENAME);
+?>
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Contact</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <?php Header::setupHeader();
+    echo "<script>var pid='" . js_escape($pid) . "'</script>";
```

Comment:
you shouldn't have the quotes around calls to js_escape (it will add them, if needed):
```
echo "<script>var pid=" . js_escape($pid) . "</script>";
```

Reply:
That was for you:)